### ISSUE TITLE
Issue with DirichletProcess

### ISSUE THREAD

**Issue by as4529** [blei-lab/edward#652]
I'm trying to reproduce the simple `DirichletProcess` example in the Model/Compositionality page of the documentation but am unable to do so – the DP samples with `alpha = 1.0` and `alpha = 50.0` look about the same.

I think the issue might be the parameter passed to `Bernoulli` in `_sample_n_body` of `DirichletProcess`. It seems to be using `probs_k` as `logits` instead of as `probs` .

**Comment by dustinvtran** [comment]
Thanks! fixed above
